ci_framework/roles/cifmw_ceph_client/defaults/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright 2020 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+
+# All variables within this role should have a prefix of "cifmw_ceph_client"
+cifmw_ceph_client_debug: "{{ (ansible_verbosity | int) >= 2 | bool }}"
+cifmw_ceph_client_hide_sensitive_logs: true
+cifmw_ceph_client_config_home: "/etc/ceph"
+tripleo_ceph_client_fetch_dir: "/tmp"
```

Comment:
we can replace `tripleo_ceph_client_fetch_dir` to `cifmw_ceph_client_fetch_dir`.

Reply:
done